docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+### Schedules
+
+A Schedule contains instructions for starting a [Workflow Execution](/workflows#workflow-execution) at specific times.
+Schedules provide a more flexible and user-friendly approach than [Temporal Cron Jobs](/workflows#temporal-cron-job).
+
+## SDK Metrics
+
+| Feature                     | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [SDK Metrics](#sdk-metrics) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+SDK Metrics track various aspects of Worker performance, such as Task Queue, Namespace, poller type, Worker type, Activity type, Workflow type, and operation.
+
+For more information on metrics see, [Temporal SDK metrics reference](/references/sdk-metrics#).
+
+## Search Attributes
+
+| Feature                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Search Attributes](#search-attributes) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Search Attribute is an indexed field used in a [List Filter](/visibility#list-filter) to filter a list of [Workflow Executions](/workflows#workflow-execution) that have the Search Attribute in their metadata.
+
+## Sessions
+
+| Feature               | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Sessions](#sessions) | Yes | No   | No         | No     | No   | No  |
+
+The Go SDK provides a support for [Worker Session](dev-guide/go/features#enable-sessions).
+This makes task routing seamless by ensuring that Activity Tasks are dispatched to the same Worker without having to handle the intricacies of Task Queue naming.
+This abstraction simplifies scenarios where consistency or state is pivotal.
+
+While Go provides built-in support, other SDKs like TypeScript and Python don't have a native concept of Sessions.
+Instead, they embrace a design pattern known as [Worker-Specific Task Queues](#worker-specific-task-queues) to achieve similar consistency guarantees.
+Under this model, all computational tasks of a Workflow are designed to execute on a single Worker.
+Although this might seem like a manual replica of the Session feature in Go, it is often the preferred way to ensure tasks are performed in a consistent environment.
+
+For instance, in scenarios where there's a need to interact with file systems—like data processing tasks or engagements with legacy structures—a sticky execution ensures that all tasks are performed in the same file context.
+In the provided example, text files are written to Worker-specific folders.
+While this serves as a demonstration, real-world applications might involve dedicated machines in a Worker cluster, each corresponding to a 'sticky' Worker, or Worker-Specific Task Queue.
+
+## Signal/Cancel External Workflow
```

Comment:
These are more workflow table-stakes features that are basically Temporal-as-a-company features and not SDK features